sampleDriver/IEvsDriverCamera.h

```diff
@@ -0,0 +1,38 @@
+#ifndef ANDROID_HARDWARE_AUTOMOTIVE_EVS_V1_1_IEVSDRIVERCAMERA_H
```

Comment:
Hi @DKlimenko, I do not see anyone using this interface. Is it really needed as part of this change?